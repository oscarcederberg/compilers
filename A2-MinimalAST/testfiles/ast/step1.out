Program
  List
    FunctionDecl ID="a"
      List
      Block
        List
    FunctionDecl ID="b"
      List
      Block
        List
