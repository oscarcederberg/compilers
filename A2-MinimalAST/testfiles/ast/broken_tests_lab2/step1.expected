Program
  List
    FunctionDecl ID="a"
      List
    FunctionDecl ID="b"
      List
